chains/Schain.cpp

```diff
@@ -1245,6 +1332,16 @@ ptr< BlockProposal > Schain::createDefaultEmptyBlockProposal( block_id _blockId
 
 void Schain::finalizeDecidedAndSignedBlock( block_id _blockId, schain_index _proposerIndex,
     const ptr< ThresholdSignature >& _thresholdSig ) {
+#ifdef BITE
```

Comment:
why is this change only applied for BITE?

Reply:
Because it is a temporary soluton for the prototype.

There will be a separate long lived worker thread that will handle finalization

Creating a new thread each time is not perfect